recipes/newrelic/apm/php/redhat.yml

```diff
@@ -376,7 +382,13 @@ install:
             #
             for i in {1..10}; do
               sleep 1
-              sudo -u $nonpriv_user php -c "${WEB_PHP_INI_DIR}/php.ini" -c "${WEB_NR_INI_DIR}/newrelic.ini" -n --ini &>/dev/null
+              #
+              #    Loop through all the detected PHP directories where newrelic.ini files
+              #    were placed.
+              #
+              for ((i=0; i<${#WEB_PHP_INI_DIR[@]}; i++)); do
+                sudo -u $nonpriv_user php -c "${WEB_PHP_INI_DIR[$i]}/php.ini" -c "${WEB_NR_INI_DIR[$i]}/newrelic.ini" -n --ini &>/dev/null
```

Comment:
Doesn't each loop run the same PHP version / binary? If someone has PHP 5.6 and 8.0 installed on the system, wouldn't this run one of them with the ini file for the other version? That ini file will point to a different `newrelic.so` and fail to load. I think we need to somehow figure out which PHP binary to use with each set of ini files.

Reply:
It loops through all the detected php dirs that the install installed in.  When the `newrelic-install.sh` is in silent mode, it installs automatically in all detected PHP directories.  So the loop goes through all the specific PHP directories that have their own distinct `php.ini` and `newrelic.ini` files and doesn't intermix the different versions  of inis.  

Yes, that's a good point about the binary.  I'm pretty sure the log file has this info, so we should be able to use it to pull the correct PHP binary.